app/javascript/components/patient/Patient.js

```diff
@@ -20,9 +21,16 @@ class Patient extends React.Component {
       expandNotes: false,
       expandArrivalNotes: false,
       expandPlannedTravelNotes: false,
+      primaryLanguageDisplayName: '',
```

Comment:
can we have these consistently named across files?

Reply:
This has kinda been refactored, so your comment doesn't really apply anymore. Gonna mark as done, but lemme know if you want more changes about this.